<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="dia_semana">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>123</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hora_dia">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>97</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>00</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculadora de Salário</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>211</width>
      <height>126</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string> Nome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_hour">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Horas trabalhadas por dia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_days">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Dias trabalhados por semana</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_anual">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Salário anual</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="s_hora">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salário por Hora:</string>
    </property>
   </widget>
   <widget class="QLabel" name="s_dia">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salário Diário:</string>
    </property>
   </widget>
   <widget class="QLabel" name="s_semana">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salário Semanal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="s_mes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salário Mensal:</string>
    </property>
   </widget>
   <widget class="QLabel" name="s_anual">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salário Anual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qualSeuSalRioAnualLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>149</y>
      <width>134</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Qual é seu salário anual?</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantasHorasVocTrabalhaLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>97</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantas horas você trabalha por dia?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="anual">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>149</y>
      <width>103</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>00000000000000000000</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="quantosDiasVocTrabalhaPorSemanaLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>123</y>
      <width>220</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantos dias você trabalha por semana?</string>
    </property>
   </widget>
   <widget class="QLabel" name="escrevaSeuNomeCompletoLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>71</y>
      <width>154</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Escreva seu nome completo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nome">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>71</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
